app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+  def index
+    @movies = Movies::UseCases::FindAll.new.call
```

Comment:
You can use `movies = Movies::Representer.new.find_all` instead of `@movies = Movies::UseCases::FindAll.new.call` :)

Reply:
That's right 👍  And then remove Movies::UseCases::FindAll class at all.